web3/net.py

```diff
@@ -12,6 +12,10 @@ def listening(self):
     def peerCount(self):
         return self.web3.manager.request_blocking("net_peerCount", [])
 
+    @property
+    def chainId(self):
+        return self.version
```

Comment:
Since the docs *claim* that `web3.net.version` is an alias of `web3.net.chainId` what do you think about either

- updating the docs to instead say that `web3.net.chainId` is an alias to `web3.net.version`
- moving the RPC call to this property, and actually making `web3.net.version` return `web3.net.chainId`

Reply:
Updated docs to reflect `chainId` as an alias of `web3.net.version` as you suggested. thanks